apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.ts

```diff
@@ -28,7 +34,29 @@ router.get(
       CourseRequestRowSchema,
     );
 
-    res.send(RequestCourse({ rows, resLocals: res.locals }));
+    let lti13Info: Lti13CourseRequestInput = null;
+    if (isEnterprise() && 'lti13_claims' in req.session) {
+      try {
+        const ltiClaim = new Lti13Claim(req);
+
+        lti13Info = {
+          'cr-firstname': ltiClaim.get('given_name') ?? '',
+          'cr-lastname': ltiClaim.get('family_name') ?? '',
+          'cr-email': ltiClaim.get('email') ?? '',
+          'cr-shortname':
+            ltiClaim.get("'https://purl.imsglobal.org/spec/lti/claim/context'.label") ?? '',
+          'cr-title':
+            ltiClaim.get("'https://purl.imsglobal.org/spec/lti/claim/context'.title") ?? '',
+          'cr-institution': res.locals.authn_institution.long_name ?? '',
+        };
+      } catch (err) {
+        // If LTI information timed out or is incorrect, don't error here.
```

Comment:
When/how would this time out?

Reply:
I'd use the word "expired" instead of "timed out", "timed out" made me think there was somehow a network request hidden here.